src/grokcore/xmlrpc/tests/__init__.py

```diff
@@ -1 +1,24 @@
-# i am a package
+##############################################################################
```

Comment:
I don't think this content is supposed to be here?

Reply:
Oops. Fixed.